package.json

```diff
@@ -178,6 +179,7 @@
     "@web3-react/walletconnect": "8.1.2-beta.0",
     "array.prototype.flat": "^1.2.4",
     "array.prototype.flatmap": "^1.2.4",
+    "babel-plugin-istanbul": "^6.1.1",
```

Comment:
what's up with this babel plugin?

Reply:
this is what actually does the instrumentation: https://docs.cypress.io/guides/tooling/code-coverage#Using-code-transpilation-pipeline